src/Sdk/Cerbos/Sdk/Cloud/V1/Store/ReplaceFilesRequest.cs

```diff
@@ -0,0 +1,170 @@
+// Copyright 2021-2025 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using Google.Protobuf.Collections;
+
+namespace Cerbos.Sdk.Cloud.V1.Store
+{
+    public sealed class ReplaceFilesRequest
+    {
+        private string StoreId { get; set; }
+        private Types.Condition Condition { get; set; }
+        private Api.Cloud.V1.Store.ReplaceFilesRequest.ContentsOneofCase ContentsOneOf = Api.Cloud.V1.Store.ReplaceFilesRequest.ContentsOneofCase.None;
+        private Types.Files Files { get; set; }
+        private byte[] ZippedContents { get; set; }
+        private ChangeDetails ChangeDetails { get; set; }
+
+        private ReplaceFilesRequest() { }
+
+        public static ReplaceFilesRequest NewInstance()
+        {
+            return new ReplaceFilesRequest();
+        }
+
+        public ReplaceFilesRequest WithStoreId(string storeId)
+        {
+            StoreId = storeId;
+            return this;
+        }
+
+        public ReplaceFilesRequest WithCondition(Types.Condition condition)
+        {
+            Condition = condition;
+            return this;
+        }
+
+        public ReplaceFilesRequest ContentsZippedContents(byte[] zippedContents)
+        {
+            ContentsOneOf = Api.Cloud.V1.Store.ReplaceFilesRequest.ContentsOneofCase.ZippedContents;
+            ZippedContents = zippedContents;
+            return this;
+        }
+
+        public ReplaceFilesRequest ContentsFiles(byte[] zippedContents)
```

Comment:
This is a copy-paste error because it should be taking a list of `File` objects as the argument. Also, why not name it `WithFiles`?